src/agri_ai/tools/agricultural_tools.py

```
@@ -142,27 +146,34 @@ async def _arun(self, field_name: str) -> str:
+            # Get recent material usage if available
+            try:
+                material_usage = await self.agri_db.get_recent_material_usage(field_name)
+                if material_usage:
+                    status_info.append("最近の資材使用:")
+                    for usage in material_usage[-3:]:  # Last 3 usages
+                        date = usage.get("使用日", "N/A")
+                        material = usage.get("資材名", "N/A")
+                        amount = usage.get("使用量", "N/A")
+                        unit = usage.get("単位", "")
+                        status_info.append(f"  - {date}: {material} {amount}{unit}")
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)